test/e2e/cluster_upgrade_runtimesdk_test.go

```diff
@@ -26,7 +26,7 @@ import (
 	"k8s.io/utils/pointer"
 )
 
-var _ = Describe("When upgrading a workload cluster using ClusterClass with RuntimeSDK [PR-Informing] [ClusterClass]", func() {
+var _ = Describe("When upgrading a workload cluster using ClusterClass with RuntimeSDK [PR-Informing] [ClusterClass] [PR-Blocking]", func() {
```

Comment:
This change is only to get quick test signal on the PR. The change will be dropped before merging.

Reply:
Up to you, but in my opinion you can get rid of it already as the test is run in `pull-cluster-api-e2e-informing-main` anyway (which is also always triggered like e2e-main)